### PR TITLE
Refine English Doc of cond API

### PR DESCRIPTION
**Pull request by zhhsplendid**
Screenshot:

image

### CODE REVIEW

File: python/paddle/fluid/layers/control_flow.py [hunk 1]
```diff
@@ -1760,24 +1760,30 @@ def cond(pred, true_fn=None, false_fn=None, name=None):
     list of tensors.
     
     Note: 
-        The tuples or lists in ``true_fn`` and ``false_fn`` must have same
-        shape because of dataflow model of PaddlePaddle while the tensors in the
-        tuples or the lists can have different shapes.
+        The tuples or lists returned by ``true_fn`` and ``false_fn`` must have
+        same shape because of dataflow model of PaddlePaddle while the tensors
```
Comment (liym27) [inline python/paddle/fluid/layers/control_flow.py]: have same shape 建议改成 have the same shape

**Review by liym27 — DISMISSED** [review]

File: python/paddle/fluid/layers/control_flow.py (hunk 1 above)
Comment (zhhsplendid) [reply python/paddle/fluid/layers/control_flow.py]: Done

**Comment by zhhsplendid** [comment]
Xiang Lan commented, we should also add notice that all operations out of cond will be executed, it is not lazier semantics. I will modify it in this PR

Done

**Review by saxon-zh — APPROVED** [review]

**Review by lanxianghit — APPROVED** [review]
LGTM

**Pull request merged**